niviz_rater/index.py

```diff
@@ -111,31 +113,25 @@ def build_qc_entities(self, image_list):
         return qc_entities
 
 
-def get_qc_bidsfiles(qc_dataset, qc_config, bids_config):
```

Comment:
Why remove this function? It seems useful

Reply:
Ahh yes, so the main issue that I felt was plaguing that function was the `add_config_paths` step which mutates the `bids.config` module as a side-effect.

I wasn't a fan of that being a side-effect hiding under a `get` type of operation (i.e being used in other contexts for just fetching and modifying the behaviour of the entire module unintentionally)

My preference would be to maintain the signature and have the config be injected into the creation of `BIDSLayout`, but unfortunately `pyBIDS` doesn't allow for that.

Overall I thought the lesser-evil was to prevent that by removing that function.

However a good compromise could be the following:

```
def get_bids_file(dataset, bids_spec, bids_config_keys):
   layout = BIDSLayout(..., config=bids_config_keys)
   return layout.get(**bids_spec)
```

The only problem is that you'd have to be aware of how the `bids.config` module was mutated (i.e does a `bids_config_key` exist in the `bids.config` module?). 

But that's a larger issue that's internal to how pyBIDS was designed...

Lmk what you think!

Reply:
*nod* Keeping it as you have it here seems fine then. I might separate the bids file configuration update from the database creation/configuration step in my PR though :)